README.md

```diff
@@ -106,6 +109,13 @@ Or install it yourself as:
 gem install bitstyles
 ```
 
+### Install an unpublished version
+If you need to install a version of bitstyles that’s so far unpublished (e.g. there’s a bugfix or feature that’s not in a published package yet), you can use the following:
```

Comment:
Sometimes you add blank lines after a heading (which I think is more readable) and sometimes you don't. Might be worth making that consistent.